src/main/java/ch/njol/skript/events/EvtServerTime.java

```diff
@@ -0,0 +1,147 @@
+package ch.njol.skript.events;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.SkriptEventHandler;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptEvent;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.util.Time;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.*;
+
+public class EvtServerTime extends SkriptEvent {
+
+	public static class ServerTimeEvent extends Event {
+		@Override
+		public @NotNull HandlerList getHandlers() {
+			throw new IllegalStateException();
+		}
+	}
+
+	private static final long HOUR_12_MILLISECONDS = 43200000;
+	private static final long HOUR_24_MILLISECONDS = HOUR_12_MILLISECONDS * 2;
+	private static final Timer timer;
+
+	static {
+		Skript.registerEvent("Server Time", EvtServerTime.class, ServerTimeEvent.class,
+			"(server|real) time (of|at) %times%")
```

Comment:
```suggestion
			"real time (of|at) %times%")
```
my initial thought about server time is the minecraft time of the server or the length that the server has been online. this is less confusing